src/components/reports/TableScoreTag.vue

```diff
@@ -78,6 +78,7 @@ function handleToolTip(_taskId, _toolTip, _colData) {
   // if score exists
   if (
     _colData.scores?.[_taskId]?.rawScore ||
+    _colData.scores?.[_taskId]?.rawScore === 0 ||
```

Comment:
If I am understanding this correctly, could we reduce an extra conditional if we just replace the first conditional with if `...?.rawScore != undefined`?

Reply:
yes, good catch!